component/pom.xml

```diff
@@ -167,17 +163,17 @@
                         <Bundle-Name>${project.artifactId}</Bundle-Name>
                         <Export-Package>
                             io.siddhi.extension.io.grpc.*,
-                            org.wso2.grpc.*
+                            io.grpc.*,
```

Comment:
Why are we exporting the private packages

Reply:
for generic service grpc user needs to install their own jars which will have io.grpc dependencies. We need to expose them